src/main/java/org/opentripplanner/model/plan/Itinerary.java

```diff
@@ -64,12 +64,14 @@ public class Itinerary implements ItinerarySortKey {
   /* other properties */
 
   private final List<SystemNotice> systemNotices = new ArrayList<>();
+  private final boolean isSearchWindowAware;
```

Comment:
```suggestion
  private final boolean searchWindowAware;
```
The Java convention for properties is to use `is/has` for boolean accessor and without for the property name.

Reply:
Changed.